src/widgets/general.widget.tsx

```diff
@@ -44,13 +44,17 @@ export const GeneralWidget: FC<IProps> = observer(({style}) => {
           </Text>
         </>
       )}
-      <View style={tw`flex-1`} />
-      <Text style={tw`w-48 text-right`}>
-        <Text style={tw`text-xs font-bold`}>{store.ui.currentTemp}°</Text>{' '}
-        <Text style={tw`text-xs dark:text-gray-400 text-gray-600`}>
-          · {store.ui.nextHourForecast}
-        </Text>
-      </Text>
+      {!!store.ui.weatherApiKey && (
```

Comment:
Don't tie it to a key, better only to show it when some value is there (change it to check if currentTemp is not null)